packages/common/src/GRPCClient.ts

```diff
@@ -757,6 +757,82 @@ export default class ConcordiumNodeClient {
         const branch = await this.client.getBranches(v2.Empty).response;
         return translate.branch(branch);
     }
+
+    /**
+     * Get information related to the baker election for a particular block.
+     *
+     * @param blockHash an optional block hash to get the instance states at, otherwise retrieves from last finalized block.
```

Comment:
```suggestion
     * @param blockHash an optional block hash to get the election info at, otherwise retrieves from last finalized block.
```
:upside_down_face: